src/index.ts

```diff
@@ -158,6 +159,19 @@ export class AllbridgeCoreSdk {
     return await this.bridgeService.approve(provider, approveData);
   }
 
+  /**
+   * Get token balance
+   * @param params
+   * @param provider
+   * @returns Token balance
```

Comment:
Integer value or decimal? I think it should be specified in the doc.

Reply:
<img width="948" alt="image" src="https://user-images.githubusercontent.com/107694759/203326186-dc2971f0-8aca-4e9f-b655-3b592bb8653a.png">

Reply:
looks fine, I think, from the point of docs